iup/get_projections.py

```diff
@@ -198,12 +198,17 @@ def get_projection(
 
         raw_data = raw_data.with_columns(
                 daily = pl.lit(fitted_daily)
+        ).with_columns(
+            cumulative = pl.col('cumulative') + pl.col('previous')
         )
 
         pred_data = pl.concat([pred_data, 
                             raw_data],
                                 how = 'vertical')
 
+    pred_data = pred_data.select(
```

Comment:
Try chaining this with the above! the `x = x.something()` paradigm is much harder to reason about than a chain like `pred_data = pl.concat([whatever]).select([whatever])`